src/tests/pam_test_client.c

```diff
@@ -49,10 +49,10 @@ static struct pam_conv conv = {
     NULL
 };
 #else
-static int dummy_pam_conv(int num_msg,
-                          const struct pam_message **msgm,
-                          struct pam_response **response,
-                          void *appdata_ptr)
+int dummy_pam_conv(int num_msg,
+                   const struct pam_message **msgm,
```

Comment:
removing const fixed the warning for me...if it was just a single * it would work but there's some magic if it's **...I'm too tired to check now, sorry,,